### PR TITLE
ping, ping6: add Spanish translation

### PR DESCRIPTION
**Pull request by Jaimepas77**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/ping.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ping
+
+> Enviar paquetes ICMP ECHO_REQUEST ("pings") a hosts de la red.
+> Más información: <https://manned.org/ping>.
+
+- Enviar pings a un host:
```
Comment (navarroaxel) [inline pages.es/common/ping.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Envia pings a un host:
```

File: pages.es/common/ping.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping {{host}}`
+
+- Enviar un número determinado de pings a un host:
+
+`ping -c {{numero}} {{host}}`
+
+- Enviar pings a un host especificando el intervalo de tiempo entre peticiones (por defecto un segundo):
```
Comment (navarroaxel) [inline pages.es/common/ping.md:14]:
```suggestion
- Envia pings a un host especificando el intervalo de tiempo entre peticiones (por defecto es 1 segundo):
```

File: pages.es/common/ping.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -i {{segundos}} {{host}}`
+
+- Enviar pings a un host sin intentar resolver nombres simbólicos de direcciones:
```
Comment (navarroaxel) [inline pages.es/common/ping.md:18]:
```suggestion
- Envia pings a un host sin intentar resolver nombres simbólicos de direcciones:
```

File: pages.es/common/ping.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -n {{host}}`
+
+- Enviar pings a un host y emitir un sonido cuando un paquete sea recibido (si la terminal lo soporta):
```
Comment (navarroaxel) [inline pages.es/common/ping.md:22]:
```suggestion
- Envia pings a un host y emite un sonido cuando un paquete es recibido (si la terminal lo soporta):
```

File: pages.es/common/ping.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`ping -a {{host}}`
+
+- Mostrar también un mensaje si no se recibió respuesta tras enviar un ping:
```
Comment (navarroaxel) [inline pages.es/common/ping.md:26]:
```suggestion
- Muestra también un mensaje si no se recibió respuesta:
```

File: pages.es/common/ping6.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# ping6
+
+> Enviar paquetes ICMP ECHO_REQUEST ("pings") a hosts de la red usando direcciones IPv6.
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:3]:
```suggestion
> Envia paquetes ICMP ECHO_REQUEST (pings) a hosts de la red usando direcciones IPv6.
```

File: pages.es/common/ping6.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+> Más información: <https://manned.org/ping6>.
+
+- Enviar pings a un host:
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:6]:
```suggestion
- Envia pings a un host:
```

File: pages.es/common/ping6.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`ping6 {{host}}`
+
+- Enviar un número determinado de pings a un host:
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:10]:
```suggestion
- Envia un número específico de pings a un host:
```

File: pages.es/common/ping6.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`ping6 -c {{numero}} {{host}}`
+
+- Enviar pings a un host especificando el intervalo de tiempo entre peticiones (por defecto un segundo):
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:14]:
```suggestion
- Envia pings a un host, especificando el intervalo de tiempo entre peticiones (por defecto es 1 segundo):
```

File: pages.es/common/ping6.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+`ping6 -i {{segundos}} {{host}}`
+
+- Enviar pings a un host sin intentar resolver nombres simbólicos de direcciones:
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:18]:
```suggestion
- Envia pings a un host sin intentar resolver nombres simbólicos de direcciones:
```

File: pages.es/common/ping6.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`ping6 -n {{host}}`
+
+- Enviar pings a un host y emitir un sonido cuando un paquete sea recibido (si la terminal lo soporta):
```
Comment (navarroaxel) [inline pages.es/common/ping6.md:22]:
```suggestion
- Envia pings a un host y emite un sonido cuando un paquete es recibido (si la terminal lo soporta):
```

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**